### DISCUSSION TITLE
Using Livewire with Breeze causes navigation and file uploading issue on production application

### DISCUSSION THREAD

**Discussion by Merchant6** [livewire/livewire#8829] [1. Help]
I am using Breeze scaffolding with Livewire installed, whenever I publish Livewire assets the Breeze navigation links redirects to localhost URL in a production environment, deleting the published assets resolves the issue.

Secondly inside the Breeze App I have created a File uploading feature using Livewire, which is not firing the AJAX requests and just refreshes the page. Without publishing the assets and config the feature does nothing and gives a console error that _404 livewire.js not found_, but after publishing the config file and giving the asset URL, the console error vanishes and the file uploading issue still persists. 

`PHP: 8.2.23`
`Laravel Framework: 11.22.0`
`Livewire: v3.5.6`
`Laravel Breeze: v2.1.4`

**Comment by Fulue** [comment]
Check your .env file for correct values in the APP_URL field. Make sure it is set to your production domain (not localhost).
In config/livewire.php, ensure that the asset_url and app_url are correctly set.

**Reply by Merchant6** [reply]
Thanks for reply, in the .env file the APP_URL is indeed the production domain and in the config/livewire.php my app_url and asset_url are set to the production domain as well.
